apps/src/music/views/MusicView.jsx

```diff
@@ -195,6 +211,15 @@ class UnconnectedMusicView extends React.Component {
     ) {
       this.musicBlocklyWorkspace.selectBlock(this.props.selectedBlockId);
     }
+
+    if (
+      !isEqual(
```

Comment:
I think we've used JSON serialization to do deep compares elsewhere in this lab.  Is this a similar situation?  Should we settle on one comparison technique?

Reply:
Fair point, I'm good to stick with JSON serialization for consistency! I know it's used in the (currently unused) tracks timeline view

Reply:
Updated to use JSON serialization